Modules/Sources/PointOfSale/Presentation/Orders/POSOrderDetailsView.swift

```diff
@@ -377,59 +376,98 @@ private extension POSOrderDetailsView {
 
 // MARK: - Actions
 private extension POSOrderDetailsView {
-    enum POSOrderDetailsAction: Identifiable, CaseIterable {
+    enum POSAction: Identifiable, CaseIterable {
+        case issueRefund
         case emailReceipt
 
         var id: String { title }
 
         var title: String {
             switch self {
-            case .emailReceipt:
-                Localization.emailReceiptActionTitle
+            case .issueRefund:  Localization.issueRefundActionTitle
+            case .emailReceipt: Localization.emailReceiptActionTitle
+            }
+        }
+
+        var accessibilityHint: String {
+            switch self {
+            case .issueRefund:  Localization.issueRefundAccessibilityHint
+            case .emailReceipt: Localization.emailReceiptAccessibilityHint
+            }
+        }
+
+        var priority: Int {
+            switch self {
+            case .issueRefund:  100
+            case .emailReceipt: 50
             }
         }
 
-        func available(for order: POSOrder) -> Bool {
+        func isAvailable(for order: POSOrder, flags: POSFeatureFlagProviding) -> Bool {
+            guard order.status == .completed else { return false }
```

Comment:
This check breaks the current behavior of showing the "send receipt" button for refunded orders, since will return early if the order is not completed. It should show like this:

<img width="2266" height="1488" alt="Simulator Screenshot - iPad mini (A17 Pro) - US store - 2025-12-03 at 07 47 52" src="https://github.com/user-attachments/assets/c35cb6e7-f925-4a44-a644-6af9b77b4098" />